PyQt5-stubs/QtCore.pyi

```diff
@@ -2755,13 +2761,19 @@ class Qt(sip.simplewrapper):
         def __invert__(self) -> 'Qt.Alignment': ...
         def __index__(self) -> int: ...
         def __int__(self) -> int: ...
+        def __or__ (self, other: typing.Union['Qt.Alignemtn', 'Qt.AlignmentFlag', int]) -> 'Qt.Alignemtn': ...
+        def __and__(self, other: typing.Union['Qt.Alignemtn', 'Qt.AlignmentFlag', int]) -> 'Qt.Alignemtn': ...
+        def __xor__(self, other: typing.Union['Qt.Alignemtn', 'Qt.AlignmentFlag', int]) -> 'Qt.Alignemtn': ...
+        def __ror__ (self, other: 'Qt.AlignmentFlag') -> 'Qt.Alignemtn': ...
+        def __rand__(self, other: 'Qt.AlignmentFlag') -> 'Qt.Alignemtn': ...
+        def __rxor__(self, other: 'Qt.AlignmentFlag') -> 'Qt.Alignemtn': ...
```

Comment:
```suggestion
        def __or__ (self, other: typing.Union['Qt.Alignment', 'Qt.AlignmentFlag', int]) -> 'Qt.Alignment': ...
        def __and__(self, other: typing.Union['Qt.Alignment', 'Qt.AlignmentFlag', int]) -> 'Qt.Alignment': ...
        def __xor__(self, other: typing.Union['Qt.Alignment', 'Qt.AlignmentFlag', int]) -> 'Qt.Alignment': ...
        def __ror__ (self, other: 'Qt.AlignmentFlag') -> 'Qt.Alignment': ...
        def __rand__(self, other: 'Qt.AlignmentFlag') -> 'Qt.Alignment': ...
        def __rxor__(self, other: 'Qt.AlignmentFlag') -> 'Qt.Alignment': ...
```

Reply:
I though I had fixed this one already. I think I was waiting on CI to see if it could pick it up